infrastructure/deploy.sh

```diff
@@ -138,6 +138,11 @@ done
 # Copy ingress config to top level so it can be applied.
 cp deploy/ci_ingress.tf .
 
+# If this file still exists, the previous deploy didn't succeed, so recover
+if [ -f nomad-configuration/client-instance-user-data.tpl.sh.bak ]; then
+    mv nomad-configuration/client-instance-user-data.tpl.sh.bak nomad-configuration/client-instance-user-data.tpl.sh
+fi
```

Comment:
I'm confused by this. What case is this for? Why not just remake `nomad-configuration/client-instance-user-data.tpl.sh` anyway?